nemo/collections/asr/models/ctc_models.py

```diff
@@ -823,13 +823,6 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
 
         results.append(model)
 
-        model = PretrainedModelInfo(
```

Comment:
Why is this being removed?

Reply:
Because it is duplicated. Please expand the code block you will see it is duplicated.